bin/impl/commands.sh

```diff
@@ -165,8 +165,10 @@ function uno_start_main() {
       if [[ -z $tmp ]]; then
         if [[ $ACCUMULO_VERSION =~ ^1\..*$ ]]; then
           "$ACCUMULO_HOME"/bin/start-all.sh
-        else
+        elif [[ $ACCUMULO_VERSION =~ ^2\.[0-1]\.[0-3]*$ ]]; then
```

Comment:
This would match on 2.1.10, but 2.1.4 will have the `--local` flag if my PR is merged in the main repo to backport it to 2.1.4. So this can just be:

```suggestion
        elif [[ $ACCUMULO_VERSION =~ ^2\.[0-1]\.[0-3]$ ]]; then
```